### PR TITLE
[BOP-178] doc changes to support manifest

### PR DESCRIPTION
**Pull request by sakshisharma84**
JIRA Ticket: https://mirantis.jira.com/browse/BOP-178

bctl now supports creating addons via manifest.

We can create AddOns via HelmCharts as well as Manifests. For this, you can specify the kind of addon you want to create in  the AddOnSpec. It can be set to either `manifest` or `chart`.

**AddOn with kind : chart**

```
addons:
      - name: my-grafana
        kind: chart
        enabled: true
        namespace: monitoring
        chart:
          name: grafana
          repo: https://grafana.github.io/helm-charts
          version: 6.58.7
          values: |
            ingress:
              enabled: true 
```

**AddOn with kind : manifest**

```
addons:
      - name: calico
        kind: manifest
        enabled: true
        namespace: boundless-system
        manifest:
          url: "https://raw.githubusercontent.com/projectcalico/calico/v3.26.3/manifests/calico.yaml" 
```

### CODE REVIEW

**Review by nwneisen — CHANGES_REQUESTED** [review]
Can you add a section as `website/content/docs/blueprints/addons/manifests.md` that gives information on using manifests?

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ It is currently possible to view and edit the documentation website locally. See
 4. Add wordpress addon to the `blueprint.yaml`:
    ```YAML
    - name: wordpress
-     kind: helm
+     kind: chart
```
Comment (ranyodh) [inline README.md:51]: **question:** I am sure we discussed this before, but I completely forgot about it. 
What is the reason to change `helm` > `chart`?

Comment (sakshisharma84) [reply README.md:51]: We always supported chart and not helm. https://github.com/Mirantis/boundless-operator/pull/10

**Comment by sakshisharma84** [comment]
> Can you add a section as `website/content/docs/blueprints/addons/manifests.md` that gives information on using manifests?

@nwneisen  There is already one. 
https://github.com/Mirantis/boundless/blob/main/website/content/docs/blueprints/addons/manifest.md

File: README.md (hunk 1 above)
Comment (sakshisharma84) [reply README.md:51]: And if it makes more sense, this maybe something we can fix later in both boundless cli and operator.

Comment (ranyodh) [reply README.md:51]: The existing documentation have `kind: helm`. Well, the fields was not being used at that time. So, it really didn't matter what we put there.

May be I can rephrase my question. What makes more sense
 - "add-on of kind `helm`" or
 -  "add-on of kind `chart`". 
 
We kinda refer them as _Helm add-ons_.

Comment (sakshisharma84) [reply README.md:51]: I get that. I think that will be a good question for @vivek and Sergey wrt user experience.

**Review by nwneisen — APPROVED** [review]
Thanks for taking a look. I'll look into why CI isn't publishing docs

**Pull request merged**